test/controllers/update/update.any.trusts.involved.spec.ts

```diff
@@ -60,13 +60,6 @@ describe("Update any trusts involved controller tests", () => {
       expect(resp.text).not.toContain(PAGE_TITLE_ERROR);
     });
 
-    test(`renders page not found if FEATURE_FLAG_ENABLE_UPDATE_TRUSTS = true`, async () => {
```

Comment:
these 2 removed tests look like they should still be used (line 63 and line 106 in github review showing previous version of this file)
test(`renders page not found if FEATURE_FLAG_ENABLE_UPDATE_TRUSTS = true`,
and
test("POST with FEATURE_FLAG_ENABLE_UPDATE_TRUSTS = true returns 404"

The flag is effectively always true now so should just need to update description and remove the mockIsActiveFeature

Reply:
tbh I'm not sure what these tests were for or what they refer to exactly as there were no changes to the update any trusts controller. I'm not sure why we'd expect 404. Are you sure we still want to use these?

Reply:
They relate to this: router.route(config.UPDATE_ANY_TRUSTS_INVOLVED_URL)
  .all(
    isFeatureDisabled(config.FEATURE_FLAG_ENABLE_UPDATE_TRUSTS)...



export const isFeatureDisabled = (featureFlagValue: string) => {
  return (req: Request, res: Response, next: NextFunction) => {
    if (!isActiveFeature(featureFlagValue)) {
      return next();
    }

    return res.status(constants.HTTP_STATUS_NOT_FOUND).render(NOT_FOUND_PAGE);
  };
};

Reply:
As it is always true should the UPDATE_ANY_TRUSTS_INVOLVED_URL be removed?

Reply:
Like the submit by paper this would also fall under another ticket.

Reply:
I think we can dispense with the tests otherwise they will just fail if we try to bring them back in.